content/contributing/project-information/release-process.md

```diff
@@ -0,0 +1,121 @@
++++
+title = "Release Process"
+insert_anchor_links = "right"
+[extra]
+weight = 3
++++
+
+Bevy uses three-months-long development cycles, delimited by a weeks-long rolling release process.
+
+This means that bevy releases a new version roughly once every three to four months. In an effort to avoid crunch and encourage sustainable development, we do not pick specific dates or deadlines.
```

Comment:
```suggestion
This means that Bevy releases a new version roughly once every three to four months. In an effort to avoid crunch and encourage sustainable development, we do not pick specific dates or deadlines.
```
Bevy is a proper noun.